projects/analogue-paths/index.md

```diff
@@ -0,0 +1,70 @@
+--- 
+authors: 
+  - Joshua Pacheco
+authors-url: 
+  - "http://incom.org/profil/7029"
+layout: project
+title: "Paths: An exploration of analogue algorithms"
+type: project
+---
+
+# PATHS
+## An exploration of analogue algorithms
+
+This work is part of a class at the University of Applied Sciences Potsdam. The project engages in analogue generative design. Initially an analogue algorithm has been written. Inspiration was supported by algorithms, Sol LeWitt uses to give his assistants to execute art works in his name. This initially analog algorithm has been extended and developed over many iterations and finally put in form.
+
+For the algorithms first iteration pencils and sheets of paper were handed to 5 people. They were instructed to consequently follow the rules of the algorithm listed below.
+
+### Iteration 1:
+Draw a rectangular form that is larger than half of the sheet. Draw another form into the first form, that is larger than half of the this first form and one that is smaller. Draw another two forms into each of the previous forms, one of them smaller and one larger than the previous form. Repeat until the sheet is full.
+
+The results turned out to be rough and did not fulfil the visual expectations.
+Therefore in the following steps the algorithm has continuously been adjusted to the impressions of natural and organic structures.
+
+![](https://raw.githubusercontent.com/josues/paths/master/assets/visuals/inspiration/Endpräsentation.004.jpeg)
```

Comment:
Don't use full URLs. These point to your repo and not the the project repo. As the Wiki states. Images should be placed in a folder called `./assets/images/` also the filenames are not url save. They should be lowercase and have no `öäü` in it. even the `.` should be removed. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#structure